src/components/custom-aggrid/custom-aggrid.tsx

```diff
@@ -43,7 +43,7 @@ const useStyles = makeStyles<GridStudyTheme, CustomAGGGridStyleProps>(
 
             // hides right border for header of "Edit" column due to column being pinned
             '& .ag-pinned-left-header': {
-                borderRight: (props) =>
+                borderRight: (props: Record<string, any>) =>
```

Comment:
why do we need to change this ? It was already well typed

Reply:
thank @archendel for the pinpoint